dungeon/src/contrib/utils/components/skill/DamageProjectile.java

```diff
@@ -242,10 +240,10 @@ public void accept(final Entity entity) {
         SkillTools.calculateLastPositionInRange(startPoint, aimedOn, projectileRange);
 
     // Calculate the velocity of the projectile
-    Point velocity = SkillTools.calculateVelocity(startPoint, targetPoint, projectileSpeed);
+    Vector2 velocity = SkillTools.calculateVelocity(startPoint, targetPoint, projectileSpeed);
 
     // Add the VelocityComponent to the projectile
-    VelocityComponent vc = new VelocityComponent(velocity.x, velocity.y, onWallHit, true);
+    VelocityComponent vc = new VelocityComponent(velocity.x(), velocity.y(), onWallHit, true);
```

Comment:
```suggestion
    VelocityComponent vc = new VelocityComponent(velocity, onWallHit, true);
```

Reply:
wäre aber eine api-ergänzung in VelocityComponent (allerdings durchaus sinnvoll in meinen augen)